### ISSUE TITLE
Some glyphs or combinations of glyphs have weird rendering

### ISSUE THREAD

**Issue by Corne2Plum3** [face-hh/griddycode#62]
**Describe the bug**
Some glyph of combinations of the font have weird rendering.

* Digits 0, 6 and 9 aren't rendered properly, there are "small holes" in the font
* Arrows (`->`, `-->`, etc.) aren't perfectly straight...
* `/*` and `*/` is completely broken.

Seems like the theme or the text color doesn't change anything.

**To Reproduce**
Type some text as described above on any file.

**Expected behavior**
The text is rendered properly

![image](https://github.com/face-hh/griddycode/assets/57914482/178a123f-0bc6-4c36-9d9c-92b127b6f7f4)

**Screenshots**
![Screenshot_20240324_171609](https://github.com/face-hh/griddycode/assets/57914482/e35e6b3d-8f5e-411a-9577-d7ea2b72c5b2)

**Desktop (please complete the following information):**
 - OS: 
 ```
OS: Ubuntu 23.10 x86_64 
Kernel: 6.5.0-26-generic 
Shell: bash 5.2.15 
Resolution: 1920x1080 
 DE: Plasma 5.27.8 
WM: kwin 
WM Theme: Flight-Color-Dark-Aurorae 
CPU: 12th Gen Intel i7-12650H (16) @ 4.600GHz 
GPU: NVIDIA GeForce RTX 4070 Max-Q / Mobile 
GPU: Intel Alder Lake-P GT1 [UHD Graphics] 
Memory: 15686MiB 
 ```
 - Version 1.1.1

**Comment by face-hh** [comment]
They're called "ligatures" and come from FiraCode. It's intended, and the only way to "fix" it would be to add a font picker, which isn't there yet

**Comment by zSnails** [comment]
There is a bug with the built in fira code font, I checked using the one on my system and everything is rendered fine, this could be fixed by #66.

We can also update the built in font, as that's the one with the issues.
